src/main/clojure/aerospike_clj/mock_client.clj

```diff
@@ -1,12 +1,25 @@
 (ns aerospike-clj.mock-client
   (:refer-clojure :exclude [update])
   (:require [promesa.core :as p]
+            [aerospike-clj.client :as client]
             [aerospike-clj.protocols :as pt]
             [aerospike-clj.utils])
-  (:import (com.aerospike.client AerospikeException ResultCode)))
+  (:import (com.aerospike.client AerospikeException ResultCode)
+           (java.time Instant)))
 
 (def ^:private DEFAULT_SET "__DEFAULT__")
 
+(defn now-epoch
+  "Mocks `now` in unix epoch for TTL usages.
+  Returns the epoch time of 01/01/2022. Can be redefined for using a different `now`"
```

Comment:
Why not use Aerospike epoch, i.e. `2010-01-01T00:00:00.000Z`?

Reply:
Because that would create a special case where TTL==expiration. 
Library users will not be able to use the mock to test/validate that their TTL logic works correctly.